sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClient.cs

```diff
@@ -1501,6 +1509,13 @@ internal HttpMessage CreateGetConfigurationSettingsRequest(string key, string la
             {
                 uri.AppendQuery("snapshot", snapshot, true);
             }
+            if (tags != null && !(tags is ChangeTrackingList<string> changeTrackingList0 && changeTrackingList0.IsUndefined))
+            {
+                foreach (var param in tags)
+                {
+                    uri.AppendQuery("tags", param, true);
```

Comment:
could you call the last parameter by name? I think it's important as it makes this security aspect of this code clearer.

Reply:
Hmm, this change actually came from the generator. Can you elaborate and what exactly I should change in this PR ?

Reply:
`uri.AppendQuery("tags", param, escapeValue : true);`

Reply:
okay I see what you're saying. Given that this change is generated, I can open an issue in autorest to see if this tweak can be made ? Unless I'm missing some customization that can be made to achieve this in this PR ? 😃

Reply:
I agree with Jorge on this one.   It's a good request for CodeGen but out of scope for this PR as AppConfig does not have direct control over this.   
@jorgerangel-msft:  Would you mind filing an issue in the AutoRest C# repository?

Reply:
Opened https://github.com/Azure/autorest.csharp/issues/4807.